db/migrate/20170716191202_add_hide_notifications_to_mute.rb

```diff
@@ -0,0 +1,5 @@
+class AddHideNotificationsToMute < ActiveRecord::Migration[5.1]
+  def change
+    add_column :mutes, :hide_notifications, :boolean, default: false, null: false
```

Comment:
You should set `default: true` to let all existing mutes to hide notifications.

Reply:
Actually, this should be caught by strong_migrations.. Does that gem not work? Adding a not-null column is a blocking operation. We recently added some helper methods from gitlab to use when an operation like this must be performed